csrc/scheduler/pointwise.cpp

```diff
@@ -524,6 +524,16 @@ std::unique_ptr<PointwiseParams> getPointwiseHeuristics(
       divisible_split,
       vectorizable_inputs_outputs_entry.get());
 
+  // Check if fusion has BlockQuantizationOp
+  auto fusion_has_block_quantization =
+      ir_utils::getOpsOfType<BlockQuantizationOp>(fusion).size() > 0;
+
+  // Limit unroll factor for fusions with BlockQuantizationOp. The runtime
+  // function which implements quantization assumes no unrolling
+  if (fusion_has_block_quantization && unroll_factor > 1) {
+    unroll_factor = 1;
```

Comment:
Is this change always safe? I don't remember if the unroll factor is just an optional parameter that can be any value. For example, let's say this changes reduces the unroll factor from, say, 4, to 1. Would that possibly mean increasing blockDim.x by a factor of 4? If so, are we sure the increased blockDim.x is still with the hardware limit?